### PR TITLE
add comare_workflow_task sample to monitoring/app

### PR DESCRIPTION
**Pull request by o-mura**
- add comare_workflow_task sample to monitoring/app

### CODE REVIEW

File: scenarios/monitoring/app/compare_workflow_task/compare_workflow_task.dig [hunk 1]
```diff
@@ -0,0 +1,41 @@
+_export:
+  td:
+    database: temporarly_${session_id}
```
Comment (akito19) [inline scenarios/monitoring/app/compare_workflow_task/compare_workflow_task.dig:3]: typo?

```diff
- temporarly_${session_id}
+ temporarily_${session_id}
```

Comment (akito19) [reply scenarios/monitoring/app/compare_workflow_task/compare_workflow_task.dig:3]: or `temporary_${session_id}`

Comment (o-mura) [reply scenarios/monitoring/app/compare_workflow_task/compare_workflow_task.dig:3]: Thanks!
This is typo!

**Review by akito19 — APPROVED** [review]
Thank you. Looks good to me.

**Pull request merged**